### ISSUE TITLE
Orientation of world frame and initial offset

### ISSUE THREAD

**Issue by mstoelzle** [tud-cor-sr/ros2-mocap_optitrack#9]
As far as I know, Emanuele has set the z-axis to be oriented downwards in the Optitrack Body software. This would mean, that the initial offset of 19mm should also act along the negative z-axis:

https://github.com/tud-cor-sr/ros2-mocap_optitrack/blob/master/mocap_optitrack_w2b/config/world_to_base.yaml#L10
https://github.com/tud-cor-sr/ros2-mocap_optitrack/blob/master/mocap_optitrack_w2b/src/WorldToBase.cpp#L17

**Comment by piepustina** [comment]
Yes, but I had to switch back to "y-up". It seems that switching to "z-up" it expresses only the positions in the new frame. The attitude is still represented in the original frame with the y axis pointing downward. I also tried to delete the rigid body and recreate it with the z-up option active... In any case, I created two configuration files , one for the y-up setup (that works correctly) and one for the z-up (that is not working properly but maybe in the future someone will figure out how it works...).  I also created two launch files.
